packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -556,7 +556,25 @@ const requestProcessor =
         })
         .build()
 
-module.exports = [client, ssr, renderer, clientOptional, requestProcessor]
+// TODO: can we make this one config with multiple entries? would that dedupe the dependencies?
+const extensions = (pkg.mobify?.app?.extensions || []).map((extension) => {
+    return {
+        name: 'extensions',
+        target: 'node',
+        mode,
+        entry: `${projectDir}/node_modules/${extension}/setup-server.js`,
+        output: {
+            path: `${buildDir}/extensions/${extension}`,
+            filename: 'setup-server.js',
```

Comment:
TODO: support ts files, add ts-loader

Reply:
Are you planning on doing this before we merge this PR?

Reply:
Is it just adding this section to allow Webpack to resolve both `.ts` and `.js`?

```
resolve: {
        extensions: ['.ts', '.js']
    },
```

Reply:
done!